topics/mvvmcross/index.md

```diff
@@ -0,0 +1,23 @@
+---
+aliases: monocross
```

Comment:
You could add a new line with `related: xamarin, dotnet, framework`.

Reply:
Added

Reply:
I just checked https://github.com/topics/monocross actually and it doesn't look to be in use, at least not on public repositories. Instead of `monocross`, how about `mvvmcross-plugin` as an alias, since that topic is being used?